vm/odroid/odroid.go

```diff
@@ -64,6 +64,9 @@ func ctor(cfg *vm.Config) (vm.Instance, error) {
 }
 
 func validateConfig(cfg *vm.Config) error {
+	if err := vm.ValidateConfig(cfg); err != nil {
```

Comment:
odroid does not use cfg.Bin, so the check will fail. gce also don't use cfg.Bin. If we remove the check from odroid and gce, then vm.ValidateConfig is not "generic" any more (if we add additional checks there that apply to all VM types, they won't work for odroid/gce).

I tend to just call exec.LookPath in adb/qemu/kvm implementations for now.